doc/whats_new.rst

```diff
@@ -0,0 +1,46 @@
+.. _whats_new:
+
+What's new
+==========
+
+.. currentmodule:: benchopt
+
+.. _changes_1_1:
+
+Version 1.1
+-----------
+
+Changelog
+~~~~~~~~~
+
+- New plotting functions with different optimality criteria, by `Nidham Gazagnadou`_ (:gh:`96`)
+
+- Support plotly for plotting functions, by `Thomas Moreau`_ and `Joseph Salmon`_ (:gh:`110` :gh:`111`)
```

Comment:
I would add Tanguy Lefort also who helped on the plottly elements (log scale issues).